examples/log_reg_model_builder.py

```diff
@@ -1 +1,94 @@
-# model builders are not available yet, coming soon
\ No newline at end of file
+#*******************************************************************************
+# Copyright 2014-2020 Intel Corporation
+# All Rights Reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License"), the following terms apply:
+#
+# You may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*******************************************************************************
+
+from sklearn.metrics import * 
+import daal4py as d4p
+import numpy as np
+from sklearn.datasets import load_iris
+from sklearn.datasets import load_breast_cancer
+from sklearn.linear_model import LogisticRegression
+
+def iris_with_intercept():
```

Comment:
I would suggest taking this out as unit tests. An example should clearly cover one case.